build/build.generic.cmd

```diff
@@ -0,0 +1,38 @@
+@echo off
+
+rem
+rem Parameters
```

Comment:
If someone calls `build.generic.cmd` with no argument, or the wrong nb of arguments, or some other easily detectable error,
could it be possible to display a help text ?

I feel your comments below are pretty good, they could just be `echo` to the console